api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -3041,6 +3041,9 @@ message DatabaseSessionStart {
     (gogoproto.embed) = true,
     (gogoproto.jsontag) = ""
   ];
+  // PostgresPID is the Postgres backend PID that was created for a Postgres
```

Comment:
Can I ask for some more details on the `Postgres backend PID` especially, 
- The scope of the ID within Postgres
- What type of value it contains (`uuid` or just a random chars). Looking at `web/packages/teleport/src/Audit/fixtures/index.ts` file below, looks like its just a random string.
- Is it a sharable ID or is security sensitive?

Reply:
> The scope of the ID within Postgres

I'm not sure what you mean by scope, but a brief summary of this ID: It's the PID of background process that the postgres server has started to handle the connection. Each PID can only handle one thing at a time, so for example a user might make a query like `SELECT pg_sleep(99999)` and they won't be able to do anything with that connection/PID until the statement completes (or they cancel it).

> What type of value it contains

It's always uint32

> Is it a sharable ID or is security sensitive?

It is not a sensitive value and can be shared. Postgres enforces controls around who can cancel/terminate a PID:
https://www.postgresql.org/docs/16/functions-admin.html#FUNCTIONS-ADMIN-SIGNAL
>> This is also allowed if the calling role is a member of the role whose backend is being canceled or the calling role has privileges of pg_signal_backend, however only superusers can cancel superuser backends.

There is a secret value exchanged at the start of the connection which also must be supplied to cancel a PID, but we do not log that value.

Reply:
Got it. Sorry for not being clear on "The scope of the ID within Postgres". I meant to ask where would the `ID` be used within the Postgres and for what purpose. I think you answered that with summary. 

I would add those description in the comment too. The reason I asked those was previously we logged some sensitive session IDs in the log. This one looks good.

Reply:
I added to the comment for the protobuf